src/main/java/frc/robot/subsystems/IndexerSubsystem.java

```diff
@@ -1,126 +1,133 @@
 package frc.robot.subsystems;
 
 import static com.revrobotics.CANSparkLowLevel.MotorType.kBrushless;
-import static edu.wpi.first.units.Units.Volts;
+import edu.wpi.first.units.Units.Volts;
```

Comment:
You need a static import for Volts.
Don't try to manage import manually, let VS Code do it for you.